src/app/modules/todo/pages/list-todo-page/list-todo-page.component.ts

```diff
@@ -43,28 +46,35 @@ export class ListTodoPageComponent implements OnInit, OnDestroy {
       .pipe(
         takeUntil(this.destroy$)
       )
-      .subscribe();
+      .subscribe((res) => res, (err) => {
+        this.showAlert(err);
+        this.authService.logout();
+      });
   }
 
   public addTodo(): void {
+    console.log(this.authService.user);
```

Comment:
if you leave this in your code, somebody else will think you are not professional ;) let go across whole your changes to make sure it's clean